cms/djangoapps/modulestore_migrator/rest_api/v1/views.py

```diff
@@ -231,35 +212,80 @@ class BulkMigrationViewSet(StatusViewSet):
         JwtAuthentication,
         SessionAuthenticationAllowInactiveUser,
     )
-    serializer_class = StatusWithModulestoreMigrationSerializer
+    serializer_class = StatusWithModulestoreMigrationsSerializer
+
+    # DELETE is not allowed, as we want to preserve all task status objects.
+    # Instead, users can POST to /api/modulestore_migrator/v1/migrations/{uuid}/cancel to cancel running tasks.
+    # GET is not allowed either. User should query /api/modulestore_migrator/v1/migrations instead.
     http_method_names = ["post"]
 
     @apidocs.schema(
         body=BulkModulestoreMigrationSerializer,
         responses={
-            201: StatusWithModulestoreMigrationSerializer,
-            401: "The requester is not authenticated.",
+            201: StatusWithModulestoreMigrationsSerializer,
+            **_error_responses,
         },
-        summary="Start a bulk modulestore to content library migration",
-        description=(
-            "Create a migration task to import multiple courses or legacy libraries "
-            "into a single content library.\n\n"
-            "**Request example**:\n\n"
-            "```json\n"
-            "{\n"
-            '  "sources": ["course-v1:edX+DemoX+2014_T1", "course-v1:edX+DemoX+2014_T2"],\n'
-            '  "target": "library-v1:org1+lib_1",\n'
-            '  "composition_level": "unit",\n'
-            '  "repeat_handling_strategy": "update",\n'
-            '  "preserve_url_slugs": true,\n'
-            '  "create_collections": true\n'
-            "}\n"
-            "```"
-        ),
     )
     def create(self, request, *args, **kwargs):
         """
-        Handle the bulk migration task creation.
+        Transfer content from multiple courses or legacy libraries into a content library.
+
+        Create a migration task to import multiple courses or legacy libraries
+        into a single content library.
```

Comment:
Nit: I'm not sure if there's any easy way to avoid this, but there's a weird line break here in the rendered docs.

Before:

<img width="631" height="102" alt="Image" src="https://github.com/user-attachments/assets/a45650c9-8be4-4bb7-acc5-4b7e2ba77e53" />

After:

<img width="566" height="118" alt="Image" src="https://github.com/user-attachments/assets/42f7ecc5-90ea-4798-96b7-8e468043a710" />

Reply:
> Nit: I'm not sure if there's any easy way to avoid this, but there's a weird line break here in the rendered docs.

hm, I'm not sure. I see it everywhere, so maybe it'd have to be fixed in https://github.com/openedx/api-doc-tools or https://github.com/axnsan12/drf-yasg. FWIW we are considering moving away from drf-yasg, so probably not worth investing time in right now.

> Also, can we tell people to refer to the "POST /modulestore_migrator/v1/migrations/" documentation for details on how this works and what the parameters are? Because they're all explained in detail there, but this endpoint comes first in the documentation.

Yeah that's a good idea.